server/routes.js

```diff
@@ -0,0 +1,8 @@
+const routes = function (app, postManager, userManager) {
+	app.post('/add_post', function (req, res) {
```

Comment:
Check, please, REST principles for the routes naming.

And here is the place for further splitting the file. But it may be done after the file gets bigger.